src/@types/buildTypeWeights.d.ts

```diff
@@ -1,5 +1,5 @@
 interface Fields {
-    readonly [index: string]: number | ((arg: number, type: Type) => number);
+    readonly [index: string]: number | ((arg: { [index: string]: any }) => number);
```

Comment:
FYI - this will change again with #33. The type for the arg will be `ArgumentNode[]`

Reply:
Yep I see that from reviewing #33. This actually make this things easier for the complexity algo because the `ArgumentNode[]` is accessible from the query AST.

Reply:
I'll resolve this when #33 is merged.